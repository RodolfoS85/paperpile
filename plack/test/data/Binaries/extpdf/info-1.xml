<?xml version="1.0"?>
<extpdf>
  <comment>Extpdf: Report information for PDF</comment>
  <command>INFO</command>
  <inFile>test.pdf</inFile>
</extpdf>
