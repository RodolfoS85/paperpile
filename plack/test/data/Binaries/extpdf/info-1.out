$VAR1 = {
          'ModDate' => 'D:20030401161505+01\'00\'',
          'page' => [
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    },
                    {
                      'size' => '609 793'
                    }
                  ],
          'status' => 'OK',
          'CreationDate' => 'D:20030401161505+01\'00\'',
          'numPages' => '9',
          'Producer' => 'Acrobat Distiller 5.0.5 (Windows)'
        };
